cmake/thirdparty/SetupMPI.cmake

```diff
@@ -40,35 +40,35 @@ if (ENABLE_FIND_MPI)
     # Merge found MPI info and remove duplication
     #-------------------
     # Compile flags
-    set(_c_flg ${MPI_C_${_mpi_compile_flags_suffix}})
+    set(_c_flag ${MPI_C_${_mpi_compile_flags_suffix}})
     if (_c_flag AND ENABLE_CUDA)
-        list(APPEND _mpi_compile_flags   
-                    "$<$<NOT:$<COMPILE_LANGUAGE:CUDA>>:${c_flg}>"
-                    "$<$<COMPILE_LANGUAGE:CUDA>:-Xcompiler=${_c_flg}>")
+        list(APPEND _mpi_compile_flags
+                    $<$<NOT:$<COMPILE_LANGUAGE:CUDA>>:${_c_flag}>
```

Comment:
Was this problem?  I.e. your version of cmake/compiler/nvcc didn't like the quotations marks around the generator expressions?

Reply:
Nope. That was just for consistency. There were typos, like _c_flag or c_flg instead of _c_flg.